module/actuation/FootController/src/FootController.hpp

```diff
@@ -79,8 +88,38 @@ namespace module::actuation {
             std::chrono::seconds set_gain_delay = std::chrono::seconds(0);
             /// @brief Desired gains for each servo
             std::map<std::string, double> desired_gains{};
+
+            /// @brief Whether or not to use orientation correction
+            bool correction_enabled = false;
+            /// @brief Proportional gain for torso orientation roll correction
+            double roll_p_gain = 0.0;
+            /// @brief Proportional gain for torso orientation pitch correction
+            double pitch_p_gain = 0.0;
+            /// @brief Integral gain for torso orientation roll correction
+            double roll_i_gain = 0.0;
+            /// @brief Integral gain for torso orientation pitch correction
+            double pitch_i_gain = 0.0;
+            /// @brief Antiwindup max I error
+            double max_i_error = 0.0;
+            /// @brief Derivative gain for torso orientation roll correction
+            double roll_d_gain = 0.0;
+            /// @brief Derivative gain for torso orientation pitch correction
+            double pitch_d_gain = 0.0;
         } cfg;
 
+        /// @brief Accumulates the integral of the roll error over time for use in PID control.
+        double integral_roll_error = 0;
+        /// @brief Accumulates the integral of the pitch error over time for use in PID control.
+        double integral_pitch_error = 0;
+
+        /// @brief Stores the timestamp of the last PID control update, used to compute time deltas for integral and
+        /// derivative calculations.
+        NUClear::clock::time_point last_update_time{};
+
+    public:
```

Comment:
All these functions seem like they're just used within the class, so they should be in the private section?